projects/ui-components/lf-login/lf-login.component.ts

```diff
@@ -195,9 +192,15 @@ export class LfLoginComponent implements OnChanges, OnDestroy {
       }
       else {
         try {
-          const response = await this.loginService.tokenClient.refreshAccessToken(refreshToken, this.client_id);
+          const oauthRegion = this.account_endpoints?.regionalDomain;
+          if (!oauthRegion) {
+            throw new Error('Unable to refresh. Cannot construct tokenClient.');
+          }
+          const tokenClient = new TokenClient(oauthRegion);
+          const response = await tokenClient.refreshAccessToken(refreshToken, this.client_id);
```

Comment:
it's fine to only update the access token in this case, if anything else were different, the call wouldn't work, we would get logged out and start over